config/base/params.env

```diff
@@ -1,6 +1,7 @@
-odh-model-controller=quay.io/opendatahub/odh-model-controller:fast
-caikit-tgis-image=quay.io/opendatahub/caikit-tgis-serving:fast
-caikit-standalone-image=quay.io/opendatahub/caikit-nlp:fast
-tgis-image=quay.io/opendatahub/text-generation-inference:fast
-ovms-image=quay.io/opendatahub/openvino_model_server:2024.3-release
-vllm-image=quay.io/opendatahub/vllm:fast
+odh-model-controller=quay.io/opendatahub/odh-model-controller:v0.12.0
+caikit-tgis-image=quay.io/opendatahub/caikit-tgis-serving:stable-2d9db23
+caikit-standalone-image=quay.io/opendatahub/caikit-nlp:stable-994ec60
+tgis-image=quay.io/opendatahub/text-generation-inference:stable-eba83ba
+ovms-image=quay.io/opendatahub/openvino_model_server:2024.3-release-4c8c52c	
+vllm-image=quay.io/opendatahub/vllm:stable-849f0f5
+nim-state=removed
```

Comment:
so in https://github.com/opendatahub-io/opendatahub-operator/pull/1330,  if not specify nim-state in DSC, it should pass down "managed" as default. 
but here, if not get value from Operator, the default is removed.
isnt that inconsistent? 
if the requirement from UX is, unless user explicitly set "removed" in DSC, then it is always "managed", then it should be `nim-state=managed` here, right?
 in reality, it does not matter managed/remove/ "" in params.env since it will always be overrwritten by Operator. But if run odh-model-controller without Operator,  then set "removed" here is different than "managed" also cause confusion in the code.

Reply:
@zdtsw I think the intent here was to add removed explicitly as a safety in case someone ever decides to change in params.env without taking modlemesh into consideration.